### PR TITLE
prettify demo

### PR DESCRIPTION
**Pull request by valdrinkoshi**
Now looks like this
screen shot 2016-01-29 at 10 21 31 am

### CODE REVIEW

**Comment by notwaldorf** [comment]
@valdrinkoshi What's up with those tests? :)

File: demo/index.html [hunk 1]
```diff
@@ -21,83 +21,72 @@
...
-
-    <simple-dialog id="scrolling">
-      <h2>Scrolling</h2>
-      <paper-dialog-scrollable>
-        <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
-        <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+<body unresolved class="centered">
+
+  <h3><code>PaperDialogBehavior</code> can be opened, closed, toggled. Use <code>h2</code> for the title</h3>
```
Comment (notwaldorf) [inline demo/index.html]: Maybe "An element with PaperDialogBehavior" instead?

File: test/paper-dialog-behavior.html [hunk 2]
```diff
@@ -175,10 +175,6 @@ <h2 id="header">Dialog</h2>
               done();
             });
             Polymer.dom(dialog).querySelector('test-buttons').$.dismiss.click();
-            // We don't wait too long to fail.
```
Comment (notwaldorf) [inline test/paper-dialog-behavior.html]: Why aren't these needed anymore?

File: demo/index.html (hunk 1 above)
Comment (valdrinkoshi) [inline demo/index.html]: will do

File: test/paper-dialog-behavior.html (hunk 2 above)
Comment (valdrinkoshi) [inline test/paper-dialog-behavior.html]: These were optimizations in order not to wait too long, but we already have a timeout of 10s that does it, so it's just over engineering that can be removed. Also, sometimes it takes longer than 20ms for certain browsers to perform the click... :x

**Comment by valdrinkoshi** [comment]
The new release of `iron-overlay-behavior` is causing weird behaviors on our tests...will have a look now

**Comment by valdrinkoshi** [comment]
These changes are not the cause of tests failures. If I run `wct` locally all tests are green.
The only way to reproduce it locally is moving the focus away from firefox main window (e.g. switch application, or click on the searchbar of firefox) while the tests are running.
@azakus @cdata any clue why something like this is happening on Travis?

**Comment by valdrinkoshi** [comment]
failing because of https://github.com/Polymer/web-component-tester/issues/283

**Comment by notwaldorf** [comment]
LGTM, but please keep an eye on the tests (I understand what the cause is, but we should get it fixed soon)

**Comment by valdrinkoshi** [comment]
Merging this as it's just changing the demo page & not touching functionality.

**Pull request merged**